Ginger/GingerCoreNET/ALMLib/ZephyrEnt/ZephyrEntCore.cs

```diff
@@ -215,7 +214,7 @@ public TreeNode GetTestRepositoryFolderById(int treeId)
         }
         public override bool IsServerConnected()
         {
-            return IsConnectValidation;
+            return IsConnectValidationDone;
```

Comment:
what is "IsConnectValidation" flag? it was already exist? I'm confused, it's not a new flag you just added?

Reply:
i got what happened, i viewed only last commit and not all.